fa/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="ar">
+<head>
+  <title>برای پرسیدن سوال نپرس، فقط بپرسش</title>
+  <meta property="og:title" content="برای پرسیدن سوال نپرس، فقط بپرسش" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ar" />
+  <meta property="og:locale" content="ar_TN" />
```

Comment:
Not sure if this is the correct country, correct if not

```suggestion
  <meta property="og:locale" content="fa_IR" />
```